deploy-jenkins/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+dependencies:
+  - { role: login, tags: login}
```

Comment:
Are we going to copy this across or just document that they need to be logged in on the node where they are executing this?